cli/src/commands/next.rs

```diff
@@ -128,12 +128,14 @@ pub(crate) fn cmd_next(
 
     let target_revset = if args.conflict {
         start_revset
+            .children()
             .descendants()
             .filtered(RevsetFilterPredicate::HasConflict)
             .roots()
     } else {
-        start_revset.descendants_at(args.offset).minus(&wc_revset)
-    };
+        start_revset.descendants_at(args.offset)
+    }
+    .minus(&wc_revset);
```

Comment:
I still don't undestand that code change, as in `wc_revset` never can be in the conflict revset, so could you explaint it to me?

Reply:
`wc_revset` can be in the conflict revset in the non-editing case.

Reply:
fwiw, I think `start_revset.children().minus(&wc_revset).descendants(..)..` is more correct. It just doesn't matter because `--edit` is implied if `wc` had children, and the current application order helps flatten descendants expressions.